vcpkg/users/versioning-troubleshooting.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Troubleshoot versioning and feature selection issues
+description: Troubleshooting guide for common versioning issues
+author: JavierMatosD
+ms.author: javiermat
+ms.date: 11/30/2023
+ms.prod: vcpkg
+ms.topic: troubleshooting-general
+---
+
+# Versioning troubleshooting guide
+
+This guide is intended for users experiencing issues with [versioning](./versioning.md).
+
+## Inspecting the version file for a port
+
+The following process works for vcpkg registry ports. For more detailed information, see our [registry](../maintainers/registries.md#git-registries) documentation. This process involves checking the available versions of a package in the vcpkg repository.
+
+How to inspect:
+1. Navigate to the `vcpkg/versions` directory.
+2. Locate the port folder:
+   - Within the `versions` directory, ports are organized alphabetically in folders. Find the folder corresponding to the first letter of your desired port. For example, for `fmt`, look in the `f-` folder.
+3. Open the ports version file:
+   - In the appropriate alphabetical folder, locate the JSON file for your port. For `fmt`, this would be `fmt.json.`
+
+What you will find:
+
+In the port's version file, you'll see a list of available versions with details like version numbers and corresponding [Git tree-object hashes](https://git-scm.com/book/en/v2/Git-Internals-Git-Objects#_tree_objects). This information is crucial for understanding what versions you can specify in your project's `vcpkg.json` file.
+
+How to use this information:
+- Choose a version from the list that meets your projects needs.
+- Reflect this in your projects `vpckg.json` file under the appropriate dependency.
+- After updating the manifest file, run `vcpkg install` to install newly specified version.
```

Comment:
Remove this since it makes more sense to have these instructions in the resolution steps of "non-existent version failures".